builtin/credential/radius/path_config.go

```diff
@@ -114,18 +114,17 @@ func (b *backend) pathConfigRead(
 	resp := &logical.Response{
 		Data: structs.New(cfg).Map(),
 	}
-	resp.AddWarning("Read access to this endpoint should be controlled via ACLs as it will return the configuration information as-is, including any passwords.")
+	resp.AddWarning("Read access to this endpoint should be controlled via ACLs as it will return the configuration information as-is, including any secrets.")
 	return resp, nil
 }
 
-/*
- * Retrieve a complete config entry from defaults or already configured params
- */
-func (b *backend) configCreateUpdate(d *framework.FieldData, req *logical.Request) (*ConfigEntry, error) {
+func (b *backend) pathConfigCreateUpdate(
+	req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
 
+	// Build a ConfigEntry struct out of the supplied FieldData
 	cfg, err := b.Config(req)
 	if err != nil {
-		return nil, err
+		return logical.ErrorResponse(err.Error()), nil
```

Comment:
Actually it's the same behaviour as before.
In `6e486ba9fac4b8a5adb5ee8864e710ad49d85e60`, `configCreateUpdate` would only return err from a `b.Config(req)` failure and in `pathConfigWrite`.
```
cfg, err := b.configCreateUpdate(d, req)
        if err != nil {
                return logical.ErrorResponse(err.Error()), nil
        }
```

But you are right, it makes more sense to return the err object.

I'll change it